sre_maturity.md

```diff
@@ -121,21 +121,31 @@ Chaos testing in production during the available time window while still meeting
 
 ### Security - Phase 1: The ‘Crawl’ Phase
 
-There is no need to get granular with authz policies to start with, but do start with production access restricted to a tightly scoped set of individuals.
+Security are the controls (administrative, technical, physical) that are put in place to manage risk of a company to an acceptable level.
+At this phase you are not granular with authz policies to start with, but do have production access restricted to a tightly scoped set of individuals.
```

Comment:
One of the thoughts that I have regarding this topic is there are two fundamental principles regarding authn and authz:

- Limited access: access to the service is limited to those that strictly require it.
- Minimum set of permissions: users that have access to the service have the minimum amount of permissions they need to operate the service.

Additionally:
- Access is revoked as soon as possible when a user no longer needs to. For example when the user leaves the group / company.
- All access to the system is logged.

IMO those are the fundamental pieces of security for crawl.

Reply:
:100: